src/components/subscribemodal/index.tsx

```diff
@@ -1,21 +1,20 @@
 import * as React from "react";
-import * as ReactUse from "react-use";
```

Comment:
did we remove this from the package.json itself?? that was the main thing right?

Reply:
No we have not removed it from package.json.

Reply:
No we haven't removed it from package.json.